index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <nav id="navbar">
+        <header>JS Documentation </header>
+            
+            <ul>
+                <li><a class="nav-link " href="#Introduction">Introduction</a>
+             </li>
+             <li><a class="nav-link" href="#What_you_should_already_know">What_you_should_already_know</a> </li>
+             <li> <a class="nav-link" href="#JavaScript_and_Java">JavaScript and Java</a> </li>
+             <li> <a class="nav-link" href="#Hello_world">Hello world</a> </li>
+             <li><a class="nav-link" href="#Variables">Variables</a></li>
+             <li><a class="nav-link" href="#Declaring_variables">Declaring variables</a></li>
+             <li><a class="nav-link" href="#Variable scope">Variable scope</a></li>
+             <li><a class="nav-link" href="#Global_variables">Global variables</a></li>
+             <li><a class="nav-link" href="#Constants">Constants</a></li>
+             <li><a class="nav-link" href="#Data_types">Data types</a></li>
+             <li><a class="nav-link" href="#if...else_statement">if...else statement</a></li>
+             <li><a class="nav-link" href="#while_statement">while statement</a></li>
+             <li><a class="nav-link" href="#Function_declarations">Function declarations</a></li>
+             <li><a class="nav-link" href="#Reference">Reference</a></li>
+            </ul>
```

Comment:
Please remove blank spaces. You code will be much nicer and readable without that.